src/plugins/filter/filter.js

```diff
@@ -98,11 +125,24 @@ var componentName = "wb-filter",
 				$elm.before( filterUI );
 			}
 
+			itemsObserver = new MutationObserver( function() {
+				let itemsLength = $elm.find( secSelector + settings.selector ).length,
+					itemsVisible = $elm.find( secSelector + notFilterClassSel + settings.selector + visibleSelector ).length,
+					infoElm = $( "#" + $elm.get( 0 ).id + "-info" );
+
+				if ( infoElm ) {
+					infoElm.html( infoFormater( itemsVisible, itemsLength, elm.infoText ) );
```

Comment:
May be it will be better and safer to use `textContent` instead of the `html` function

Reply:
May be we can adapt to allow the use of an `<span>` element inside to scope where the text content update are going to happen.

May be it will be easier to use a data attribute or a CSS flag to identify the field to be updated instead of replacing a placeholder in the text content.

Like:

```html
<p class="wb-fltr-info"><span data-nbitem>_NBITEM_</span> / <span data-total>_TOTAL_</span> provinces</p>
```